src/api/object-api.ts

```diff
@@ -19,7 +19,7 @@ import {
 export function keys<K>(map: ObservableMap<K, any>): ReadonlyArray<K>
 export function keys<T>(ar: IObservableArray<T>): ReadonlyArray<number>
 export function keys<T>(set: ObservableSet<T>): ReadonlyArray<T>
-export function keys<T extends Object>(obj: T): ReadonlyArray<string>
+export function keys<T extends Object>(obj: T): ReadonlyArray<PropertyKey>
```

Comment:
Looks like breaking change...

Reply:
@mayorovp Is it 🤔?

Sorry I'll need a hand in understanding why, if you have the time. I only see this as an additive change, since `PropertyKey` is a union type still containing to `string`.

Reply:
The type is wider, so should be ok